plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/refactoring/quick_fixes/AddTokenAndImportStatement.java

```diff
@@ -247,6 +253,47 @@ public void createTextEdit(ComputedInfo computedInfo) {
         }
     }
 
+    // function to get the amount of commas at the end of import string
+    private int verifyComma(String line) {
+        // TODO Auto-generated method stub
+        boolean withComma = true;
+        int commaAmount = 0;
+        int len = line.length();
+
+        if (line.endsWith(")")) {
+
+            line = line.substring(0, len - 1);
+            line = line.trim();
+            len--;
+
+            while (withComma == true) {
+                if (line.endsWith(",")) {
+                    line = line.substring(0, len - 1);
+                    len--;
+                    commaAmount++;
+                } else {
+                    withComma = false;
+                    line = line.concat(")");
```

Comment:
Why bother doing `line.concat` here? When this while finishes I don't think you use the `line` for anything else (probably some leftover).